jipdate.py

```diff
@@ -498,6 +498,20 @@ def get_header():
 
     return ("\n".join(yml_iter) + "\n\n") if yml_iter is not None else "\n"
 
+def get_editor():
+    global g_yml_config
+
+    try:
+        yml_iter = g_yml_config['text-editor']
+    except:
+        # Probably no "text-editor" section in the yml-file.
+        return True
+
+    if yml_iter == "no" or yml_iter == "NO" or yml_iter == "No":
```

Comment:
```
>>> a = "No NO nO no"
>>> a.lower()
'no no no no'
```

So, one would prefer doing like this:
```python
if yml_iter.lower() == "no":
...
```
or even change 510 to 513 with:
```python
return False if yml_iter.lower() == "no" else True
```

There is one drawback using the string "yes" and "no" and that is that `yes` and `no` works as booleans in python. So, in the config file one has to write
```yml
text-editor: "no"
```
and not
```yml
text-editor: no
```
to make the `.lower()` to work. So ... after a long answer, I think it's even better to do like this:

```patch
diff --git a/jipdate.py b/jipdate.py
index db12541..856390d 100755
--- a/jipdate.py
+++ b/jipdate.py
@@ -457,7 +457,8 @@ header:
 
           This is the status update from me for the last week.
 
-          Cheers!"""
+          Cheers!
+use-text-editor: True"""
 
     with open(config_file, 'w') as f:
         f.write(yml_cfg)
@@ -502,15 +503,12 @@ def get_editor():
     global g_yml_config
 
     try:
-        yml_iter = g_yml_config['text-editor']
+        yml_iter = g_yml_config['use-text-editor']
     except:
         # Probably no "text-editor" section in the yml-file.
         return True
 
-    if yml_iter == "no" or yml_iter == "NO" or yml_iter == "No":
-        return False
-
-    return True
+    return yml_iter
 
 ################################################################################
 # Main function
```